### PR TITLE
Migration 004: Add Prettier w/ configuration

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a follow-up to #3737 to separate it into a series of PRs following the documented bulk update procedure.  (There haven't been any signals from the author regarding a separation of the changes for over a month.)  This PR is the second of four steps, which introduces the Prettier dependency, a configuration file, and the Prettier format command added to the fix command.

I've made sure to add @ExE-Boss as a co-author to the commits -- thanks for getting the ball rolling!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,12 +35,13 @@
...
     "mdn-confluence": "^1.0.3",
     "ora": "^4.0.3",
+    "prettier": "^1.18.2",
     "yargs": "^15.0.1"
   },
   "scripts": {
     "confluence": "node ./node_modules/mdn-confluence/main/generate.es6.js --output-dir=. --bcd-module=./index.js",
     "lint": "node test/lint",
-    "fix": "node scripts/fix",
+    "fix": "node scripts/fix; prettier --write **/*.js **/*.ts **/*.md **/*.json",
```
Comment (ddbeck) [inline package.json]: Can we limit this only to `.js` and `.ts` files for now? I have reservations about the way prettier rewrites Markdown and JSON and those will be harder to review. I have clearer expectations and higher confidence in the JS changes.

Comment (ExE-Boss) [reply package.json]: All&nbsp;`*.json`&nbsp;files are&nbsp;ignored by&nbsp;default, except&nbsp;for&nbsp;`package.json` and&nbsp;files in&nbsp;the&nbsp;`schemas`&nbsp;directory. https://github.com/mdn/browser-compat-data/blob/5a688f80031e38714e5a6a865384e233707e869f/.prettierignore#L1-L3

Comment (vinyldarkscratch) [reply package.json]: Done!

Comment (ddbeck) [reply package.json]: @ExE-Boss It was the schemas in particular I was concerned about. There's some vertical white space for readability in the schemas and I'm not sure we should clobber that.

I suppose we could allow `package.json` though. I'd be OK with that.

**Comment by vinyldarkscratch** [comment]
Not a problem, script has been added!

**Comment by vinyldarkscratch** [comment]
Sounds like a plan, thank you very much!  The only PRs that would most likely be affected by this migration are all my own, as well as a couple from @ExE-Boss.  I can rebase my own PRs afterwards, so they can wait.  :wink:

**Comment by ddbeck** [comment]
@vinyldarkscratch I think the PRs open against the JS files are either pretty small or belong to you, so I'm going to merge this. I can review a migration PR this week, when you're ready. 🔀

**Pull request merged**